dom-manipulation/main.js

```diff
@@ -0,0 +1,26 @@
+const btn = document.querySelector('.hot-button');
+
+let counter = 0;
+
+const mainBtn = btn;
+
+const countDisplay = document.querySelector('.click-count');
+
+mainBtn.addEventListener('click', (e) => {
+  counter += 1;
+  countDisplay.innerHTML = `Clicks: ${counter}`;
```

Comment:
You should not use `innerHTML`, it's a vulnerability. You should be using something like `textContent` instead